data/mods/innawood/items/tool_bronzecast.json

```
@@ -0,0 +1,117 @@
+[
+  {
+    "id": "bronze_axe",
+    "type": "TOOL",
+    "name": { "str": "bronze axe", "str_pl": "bronze axes" },

```

```suggestion
    "name": { "str": "bronze axe" },
```
No need to add `s` to the end, it's done automatically.